packages/strapi-plugin-upload/controllers/Upload.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
     }
 
     const files = _.get(ctx.request.files, 'files');
-    if (_.isEmpty(files)) {
+    if (_.isEmpty(files) || files.size === 0) {
```

Comment:
Why is it needed ?

Reply:
If you send a request with an empty file it will still create a tmp File with size = 0. (This is coming from koa-body -> node-formidable)